### PR TITLE
[integration] remove CSGlobals.Extensions class

### PR DESCRIPTION
**Pull request by TaykYoku**
The result of `importlib.import_module` is a bit different than `imp.find_module`, so I think that without this fix it will work differently. I have already expressed my opinion https://github.com/DIRACGrid/DIRAC/pull/5617#discussion_r758307808.

BEGINRELEASENOTES

*Configuration
FIX: fix CSGlobals.Extensions class

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/ConfigurationSystem/Client/Helpers/CSGlobals.py [hunk 1]
```diff
@@ -72,7 +25,14 @@ def getCSExtensions():
     Return list of extensions registered in the CS
     They do not include DIRAC
     """
-    return Extensions().getCSExtensions()
+    __csExt = []
```
Comment (chrisburr) [inline src/DIRAC/ConfigurationSystem/Client/Helpers/CSGlobals.py:28]: Can we get rid of `getCSExtensions` entirely? (and mark it as deprecated in rel-v7r3)

File: src/DIRAC/TransformationSystem/scripts/dirac_production_runjoblocal.py [hunk 2]
```diff
@@ -159,11 +159,9 @@ def main():
             _downloadinputdata = False
             _jobID = switch[1]
 
-    from DIRAC.ConfigurationSystem.Client.Helpers.CSGlobals import Extensions
+    from DIRAC.ConfigurationSystem.Client.Helpers.CSGlobals import getCSExtensions
 
-    ext = Extensions()
-    _vo = ext.getCSExtensions()[0]
-    _diracPath = Extensions().getExtensionPath("DIRAC")
+    _vo = getCSExtensions()[0]
```
Comment (chrisburr) [inline src/DIRAC/TransformationSystem/scripts/dirac_production_runjoblocal.py:164]: Niavely I would expect this to be `getVO()`?

In fact `__configurePilot` seems to be overriding it with `getVO()`.

Comment (TaykYoku) [reply src/DIRAC/TransformationSystem/scripts/dirac_production_runjoblocal.py]: I tried to simplify it, but not sure I understand how to test it (3bfb6d50c400afd4f0f029bea822d47d3c19a11d)

File: src/DIRAC/ConfigurationSystem/Client/Helpers/CSGlobals.py (hunk 1 above)
Comment (TaykYoku) [reply src/DIRAC/ConfigurationSystem/Client/Helpers/CSGlobals.py]: It used:
- `DIRAC/Resources/Computing/SingularityComputingElement.py`
- `DIRAC/WorkloadManagementSystem/Agent/SiteDirector.py`

File: src/DIRAC/TransformationSystem/scripts/dirac_production_runjoblocal.py (hunk 2 above)
Comment (TaykYoku) [reply src/DIRAC/TransformationSystem/scripts/dirac_production_runjoblocal.py]: From command description:
>This method was created specifically for LHCb pilots, more info
    about othe VOs is needed to make it more general.

So this is a special command for the LHCb? Maybe to make it more universal or move it to the LHCb extension?

Comment (TaykYoku) [reply src/DIRAC/TransformationSystem/scripts/dirac_production_runjoblocal.py]: @fstagni Could you look when have a time, please?

Comment (TaykYoku) [reply src/DIRAC/TransformationSystem/scripts/dirac_production_runjoblocal.py]: @fstagni For example the following line seems not universal, or ``ce.debug.ch`` CE - OK for all cases?

>"dirac-pilot.py -S %s -l %s -C %s -N ce.debug.ch -Q default -n DIRAC.JobDebugger.ch -dd"

Comment (fstagni) [reply src/DIRAC/TransformationSystem/scripts/dirac_production_runjoblocal.py]: These values can be universal

**Review by fstagni — APPROVED** [review]

**Pull request merged**